debs/SPECS/3.11.0/wazuh-api/debian/control

```diff
@@ -8,5 +8,5 @@ Homepage: http://www.wazuh.com
 
 Package: wazuh-api
 Architecture: any
-Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.10.0), wazuh-manager (<< 3.10.1), curl
+Depends: ${shlibs:Depends}, npm, nodejs (>= 4.6.1), wazuh-manager (>= 3.11.0), wazuh-manager (<< 3.10.1), curl
```

Comment:
This line is poorly fixed. Check the versions of the wazuh-manager because right now, this line makes impossible to install the wazuh-api package.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/254/commits/774ba679fde9c278ed262bdb042edc7ef1e954fc